pages/docs/reference/mpp-share-on-all-platforms.md

```diff
@@ -0,0 +1,56 @@
+---
+type: doc
+layout: reference
+title: "Discover your project"
+---
+# Share code on all platforms
+
+If you have business logic that is common for all platforms, you don’t need to write the same code for each platform – 
+just share it in the common source set.
+
+![Code shared for all platforms]({{ url_for('asset', path='images/reference/mpp/flat-structure.png') }})
```

Comment:
This picture shows a custom source set `desktopMain`. I'm not sure it should be there, as it doesn't correspond to any platform (at least it's not clear whether or not there's a target named `desktop`).

Reply:
Removed to avoid confusion.